### ISSUE TITLE
CI/CD Workflow for React SSR canary release

### ISSUE THREAD

**Issue by daine** [cds-snc/design-gc-conception#1011]
## 📇 Task
[Prep work for React SSR canary release
](https://docs.google.com/document/d/14TwlBuSd01zNkF7srl8cL1KWP0dlMvWEiDZlrbY0F6c/edit)

Work on the github actions workflow to publish canary releases for the react-ssr package

## ✅ Definition of Done / Outcomes
- [x] Done when we can publish a canary release for the new react SSR package

## 📜 Acceptance criteria
- [x] A github workflow exists that can publish our react ssr package with a canary tag and version

## 📝 More info
- Could potentially place in the same workflow files as our other package publishing actions, but will test it on a separate file first to separate it when we need to re-run it

**. 🚫 Out of scope**
_ If possible, give precisions on what is *not* part of this work_

**Comment by daine** [comment]
Next steps once this ticket is done:

- [ ] The NextJS PR for the examples codebase
- [ ] Release messaging for the new package is sent out

**Comment by daine** [comment]
This is currently in review, here is the PR: https://github.com/cds-snc/gcds-components/pull/587
